docs/monthly/2023-11-monthly.md

```diff
@@ -0,0 +1,152 @@
+tidy up the way the [Hydra protocol
+specification](https://hydra.family/head-protocol/core-concepts/specification)
+is written itself.
+
+So far, the specification was written in LaTex and rendered to a PDF. LaTeX was
+a good choice because of its expressiveness on math expressions, but not so
+great of a choice for pull requesting changes and the entry barrier is quite
+high. The majority of documentation around Hydra and Mithril is written using
+Markdown and rendered into a webpage using a static site generator, currently
+[Docusaurus](https://docusaurus.io/). Most modern markdown renderers do support
+LaTeX-style math nowadays, even the normal [Github file
+preview](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions).
+
+Hence, we experimented with using the more standard Markdown format to write the
+specification, but still being able to use LaTeX math **and** LaTeX macros
+(which are _still_ heavily used in the spec). From this we want to have both, a
+PDF and HTML rendering.
+
+TODO GIF sebastian
+
+While not yet finished, the GIF above shows already progress being made using
+[pandoc](https://pandoc.org/) to resolve macros, producing a PDF via `xelatex`
+and Docusaurus compatible markdown. Using powerful extensions of pandoc we can
+even convert TikZ, GraphViz and Mermaid into both target formats.
+
+## Community update
+
+TODO?
+
+## Conclusion
+
+The monthly review meeting for November 2023 was held on 2023-11-27 via Google
+Meet with these [slides][slides] and the [recording][recording].
+
+We are thankful to have had again so many great demos from the numerous projects
+in and around Hydra and Mithril projects - we should really schedule it to be 1
+hour planned duration. Also, if you have not watched it already, make sure to
+check out the recording, the written report is merely expanding on a couple of
+topics but cannot render the cool demonstrations.
+
+At first look, the demonstrated projects may appear not related - we even had a
+minimal command-line wallet on the show after all. But as we went on, these
+impulses spark discussions and it becomes clear that all of these projects are
+connected.
+
+For example, the terminal based wallet "Cardaminal" as it was presented the
+first time in public this month, could itself become a mithril client for faster
+synchronizing or even light-mode using the recently published rust client
+library for mithril. Furthermore, such a minimal wallet would be a great testing
+ground for a hydra-aware wallet. The MeshJS integration also has further
+potential of Hydra specific `fetcher` and `submitter` providers.
+
+On the other hand, we also stumbled over not-so-optimistic sentiment even in
+much more mature scaling solutions like the Bitcoin Ligthning Network. [This
+tweet](https://twitter.com/udiWertheimer/status/1719122153155473492) summarizes
```

Comment:
Glad you find this piece of information useful enough to make room for in the monthly report. @pgrange will be happy 😉